<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp_connect</class>
 <widget class="QMainWindow" name="tcp_connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>45</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
font: 11pt &quot;新宋体&quot;;</string>
    </property>
    <property name="text">
     <string>88</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>39</y>
      <width>110</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>45</height>
     </size>
    </property>
    <property name="text">
     <string>服务器端口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IP">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>99</y>
      <width>110</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>45</height>
     </size>
    </property>
    <property name="text">
     <string>服务器IP：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>45</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{qproperty-alignment:AlignHCenter;}
font: 11pt &quot;新宋体&quot;;</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
